packages/nns/src/governance.canister.ts

```diff
@@ -138,8 +155,11 @@ export class GovernanceCanister {
     includeEmptyNeurons?: boolean;
   }): Promise<NeuronInfo[]> => {
     const rawRequest = fromListNeurons({ neuronIds, includeEmptyNeurons });
-    const raw_response =
-      await this.getGovernanceService(certified).list_neurons(rawRequest);
+    const useOldMethod = isNullish(includeEmptyNeurons) && certified === true;
```

Comment:
Could you add a comment on why we want to use the old method when empty and certified?

Do you need the `certified === true`, why not `&& certified`?

Reply:
Good point. Thanks!